packages/flutter/test/widgets/focus_manager_test.dart

```diff
@@ -1269,6 +1270,7 @@ void main() {
       // Since none of the focused nodes handle this event, nothing should
       // receive it.
       expect(receivedAnEvent, isEmpty);
+    // ignore: deprecated_member_use
```

Comment:
I am surprised all of these ignores were necessary. I think the analyzer ignores deprecated uses in the framework.

Reply:
Yes, it does, but these aren't in the `flutter` package, they're in the `flutter_test` package.

Reply:
Oh silly me. I assumed! :)